src/plugins/media-analysis/media-analysis-display.js

```diff
@@ -89,35 +107,48 @@ function MediaAnalysisDisplay({ content }) {
       : title;
   };
 
-  const renderChapterText = (chapter, index) => {
-    const widthInPercentage = determineChapterWidthInPercentage(index);
+  const renderChapterPointer = (chapter, index) => {
+    const widthInPercentage = determineChapterSegmentWidthInPercentage(index);
+
     return (
-      <div
-        key={chapter.key}
-        style={{ width: `${widthInPercentage}%` }}
-        className="MediaAnalysisDisplay-chapterText"
-        >
-        <Markdown>{chapter.text}</Markdown>
-      </div>
+      <Tooltip key={chapter.key} title={t('chapterPointerTooltip')} placement="bottom">
+        <div
+          style={{ width: `${widthInPercentage}%` }}
+          className={classNames('MediaAnalysisDisplay-chapterPointer', { 'is-visible': index === viewingChapterIndex })}
+          onClick={() => setViewingChapterIndex(index)}
+          />
+      </Tooltip>
     );
   };
 
   const renderChapters = () => {
-    const chapterTextsAreSet = chapters.some(chapter => chapter.text);
-
     return (
       <div className="MediaAnalysisDisplay-chapters">
-        <div className="MediaAnalysisDisplay-chapterTitles">
-          {chapters.map(renderChapterTitle)}
+        <div className="MediaAnalysisDisplay-chapterSegments">
+          {chapters.map(renderChapterSegment)}
         </div>
-        {!!chapterTextsAreSet && (
+
+        {chapters.some(chapter => chapter.text) && (
           <Fragment>
-            <div className={classNames('MediaAnalysisDisplay-chapterTexts', { 'is-expanded': areTextsExpanded })}>
-              {chapters.map(renderChapterText)}
+            <div className="MediaAnalysisDisplay-chapterPointers">
+              {chapters.map(renderChapterPointer)}
+            </div>
+            <div className="MediaAnalysisDisplay-viewedChapterInfoWrapper">
+              <div className={classNames('MediaAnalysisDisplay-viewedChapterInfo', { 'is-expanded': areTextsExpanded })}>
+                <div className="MediaAnalysisDisplay-viewedChapterInfoTitle">
+                  {chapters[viewingChapterIndex].title}
+                </div>
+                <Markdown className={classNames('MediaAnalysisDisplay-viewedChapterInfoText', { 'is-expanded': areTextsExpanded })}>
+                  {chapters[viewingChapterIndex].text}
+                </Markdown>
```

Comment:
If we switch this to something along those lines (not real final code here, also it has to be in a parent grid all of them in a single cell on top of each other):

~~~
{chapters.map((chapter, index) => (
  <div key={index.toString()} style={{ visibility: index === viewingChapterIndex ? 'visible' : 'hidden' }}>
    <Markdown>{chapter.text}</Markdown>
  </div>
))}
~~~

...then we could get rid of the scrollbars and the fixed heigth, as we would just switch the visibility but always render all section annotations.

What do you think?

Reply:
I had a similar thought, of somehow rendering all annotations so that we get the maximum height of the container and remove the need for fixed height and scrollbar, but I couldn't come up with a way to render them z-layered and not stacked. As far as I see your solution also stacks them, but maybe I am getting something wrong? 🤔

Reply:
I was able to put something together locally using a parent with `display: grid` and all children being in grid row 1 and grid column 1, so the parent takes the size of the "biggest" child. Then the children could be toggled with `visibility: visible` for the active part and `visibility: collapse` for all the others.

Reply:
Thanks for explaining what you meant. Wonderful solution! Implemented 😎